### PR TITLE
#4/elewa home lander

### PR DESCRIPTION
**Pull request by littlephillips**
# Description

This pull request is a feature creation for the elewa home page lander component. 
We worked with [@Anna-Ashton](https://github.com/Anna-Ashton)  and [@chegeian](https://github.com/chegeian)

To achieve this, we needed to:

```
- Create a standalone component 
- re-using elewa hero component
- add relevant text for the page rendered
```

Fixes issue [#4 ](https://github.com/italanta/elewa-group/issues/4)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

home-lander
home0lander-mobile

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by littlephillips** [comment]
kindly review @IanOdhiambo9

File: libs/pages/elewa/home/src/lib/components/home-lander-section/home-lander-section.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<elewa-group-elewa-hero
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/components/home-lander-section/home-lander-section.component.html:1]: You're supposed to reuse the dynamic hero component that's already created and pass the correct attributes as inputs.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Good job on the design, though mobile can be improved. Reuse the dynamic hero component that was already created.

**Comment by IanOdhiambo9** [comment]
This issue has been closed.

**Pull request closed without merging**